### PR TITLE
Add pointer to qiskit/documentation issues

### PR DESCRIPTION
**Pull request by abbycross**
### Summary

With the unification of the qiskit-ibm-runtime docs and IBM Quantum Platform docs, we want to make it clear where to open an issue, depending on what page you're on when you identify a problem/suggestion to report.

### Details and comments

Issues about documentation in the Start, Build, Transpile, Verify, Run, or Migration guides sections should be opened here: https://github.com/Qiskit/documentation/issues/new/choose

### CODE REVIEW

File: CONTRIBUTING.md
Comment (abbycross) [inline CONTRIBUTING.md:16]: I wasn't sure where to add the note in contributing.md since there isn't an explicit section about opening issues, so I added a section here. Please feel free to suggest other options...

**Comment by abbycross** [comment]
@beckykd whom should I tag to review this?

File: .github/ISSUE_TEMPLATE/config.yml
Comment (Eric-Arellano) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Maybe this?

```suggestion
    about: Open an issue about documentation in the Start, Build, Transpile, Verify, Run, or Migration guides sections of docs.quantum.ibm.com (non-API documentation)
```

**Review by kt474 — APPROVED** [review]
LGTM! Thanks @abbycross & @Eric-Arellano!

**Comment by abbycross** [comment]
Thanks @kt474! Looks like someone with write access needs to do the merge, so I'll leave that to you if you don't mind :)

**Pull request merged**
